.github/workflows/CI.yml

```
@@ -36,6 +36,8 @@ jobs:
         xamarin-mac-version: latest # specify 'latest' keyword to pick up the latest available version
         xamarin-android-version: 12 # specify full version; it is not recomended option because your pipeline can be broken suddenly in future
         xcode-version: 13 # set the latest available Xcode 13
+    - name: Prepare settings.json
+      run: cp ${{ github.workspace }}/Covid19Radar/Covid19Radar/settings.json.sample ${{ github.workspace }}/Covid19Radar/Covid19Radar/settings.json
```

[修正任意]
CI なのでどちらでも問題ないですが、サンプルファイルを残す必要はないので `cp` でなく `mv` でいいと思いました！

```suggestion
      run: mv -f ${{ github.workspace }}/Covid19Radar/Covid19Radar/settings.json.sample ${{ github.workspace }}/Covid19Radar/Covid19Radar/settings.json
```